public/locations-list-map.js

```diff
@@ -220,12 +264,33 @@ $(function() {
     window.locations = result;
     window.data_by_location = toDataByLocation(locations);
 
+    // TODO (patnelson): IMHO, every single one of these show/hide settings configured by query string should be elevated
+    //  to the module-level scope for reference.
```

Comment:
remove TODO please. I want to remove all module level variables if possible since in our setup, the module is the global scope.